README.md

```diff
@@ -7,41 +7,39 @@ The scripts should work across a number of repos that include a `Dockerfile`. Th
-* In order to identify dependent services the PR script inspects rancher-compose.yml. Dependent services are those identified from the question variables ending in 'docker_image_tag' which don't match the current service repo name. For example, the presence of a question variable named pharmacy_db_docker_image_tag in the rancher-compose file for [c2s-pharmacy-finder](https://github.com/nhsuk/nhsuk-rancher-templates/tree/master/templates/c2s-pharmacy-finder) indicates that pharmacy-db is a dependent service and that, whilst the variable connecting_to_services_docker_image_tag exists, connecting-to-services would not be a dependent service
-* The naming convention employed between GitHub, Docker and Rancher is for the docker images to be named the same as the GitHub repo with the hyphen transcribed to underscore due to limitations in rancher naming
-* The image tag deployed for each *dependent* service is that of the latest GitHub release. Releases tagged as pre-release will be ignored. This may need to be revisited when we have to make changes in two dependent repos for a given PR
+* If the Travis build has been triggered by a PR a new stack will be created within the `RANCHER_ENVIRONMENT` in Rancher. The rancher and docker templates used to build the stack are pulled from the [nhsuk/nhsuk-rancher-templates](https://github.com/nhsuk/nhsuk-rancher-templates) repo on GitHub. The specific compose files are determined by `RANCHER_TEMPLATE_NAME` and the latest subdirectory (i.e. the highest numbered). This is the (imperfect) convention used by rancher catalogs to implement template versioning.
+* Image tag variable names in the rancher-compose.yml file should conform to a naming convention ({modified repo name}_DOCKER_IMAGE_TAG where {modified repo name} is the repo name in upper case with hypens replaced by underscores). This allows the image tag for the current repo being built to be set to PR-_n_ whilst all other image tags are set the default value in rancher compose.
 * A publicly accessible URL to the stack will be published in the PR's comments
 
 ## Setup in code repo
 
 1. Add this repo as a submodule `git submodule add https://github.com/nhsuk/ci-deployment.git` (note if the submodule is added in a path other than the default the path to the scripts in `.travis.yml` will need updating to that same path)
-1. Update that repo's `.travis.yml` to include the sections included within this repo's `.travis.yml` i.e. the ability to run docker and the `after_success` section
+1. Update that repo's `.travis.yml` to include the sections included within this repo's `.travis.yml`. (see [nhsuk/gp-finder](https://github.com/nhsuk/gp-finder) as an example of a repo using the scripts).
```

Comment:
I don't think it is necessary to hand off to another repo here for an example, there should be enough information available in this repo. What happens when that repo changes?